index.html

```diff
@@ -71,48 +71,52 @@ <h4 class="tskill">Skills</h4>
     <h2>MY RECENT<br>WORK</h2>
   </div>
 
-  <div class="divider"></div>
+  <div class="divider divider2" ></div>
 
   <section class="multipost">
 
     <div class="post">
       
       <img class="img-post" src="./images/Snapshoot_Portfolio@2x.png" alt="Here goes and image">
-      <div class="post-title">Title</div>
-      <div class="post-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.  
-      </div>
-      <div class="ellipse"></div>
-      <div class="languages">
-        <p>css</p><div class="updivisor"></div>
-        <p>html</p><div class="updivisor"></div>
-        <p>Bootstrap</p><div class="updivisor"></div>
-        <p>Ruby</p>
+      <div class="textos">
+        <div class="post-title">Multi-Post Stories</div>
+        <div class="post-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.  
+        </div>
+        <div class="ellipse"></div>
+        <div class="languages">
+          <p>css</p><div class="updivisor"></div>
+          <p>html</p><div class="updivisor"></div>
+          <p>Bootstrap</p><div class="updivisor"></div>
+          <p>Ruby</p>
+        </div>
+        <button class="seebutton">See project</button>
       </div>
-      
     </div>
-    <button class="seebutton">See project</button>
+    
     
 
   </section>
 
   <section class="multipost">
 
-    <div class="post">
+    <div class="post postinverted">
       
-      <img class="img-post" src="./images/Snapshoot_Portfolio@2x.png" alt="Here goes and image">
-      <div class="post-title">Title</div>
-      <div class="post-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.  
+      <img class="img-post imginverted" src="./images/Snapshoot_Portfolio@2x.png" alt="Here goes and image">
+      <div class="textos textosinverted">
+        <div class="post-title">Multi-Post Stories</div>
+        <div class="post-desc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.  
+        </div>
+        <div class="ellipse"></div>
+        <div class="languages">
+          <p>css</p><div class="updivisor"></div>
+          <p>html</p><div class="updivisor"></div>
+          <p>Bootstrap</p><div class="updivisor"></div>
+          <p>Ruby</p>
+        </div>
+        <button class="seebutton">See project</button>
       </div>
```

Comment:
- [ ]  The space between the project picture and project description is too much and the works section which does not match with the Figma design. Please fix it so it looks similar to the Figma design :+1: 

### Figma template
![Screenshot from 2022-07-12 22-58-06](https://user-images.githubusercontent.com/6615394/178613119-eb3a9b0c-b6e5-4b6a-a9b5-909c181df38c.png)

### Your project
![Screenshot from 2022-07-12 22-57-48](https://user-images.githubusercontent.com/6615394/178613122-5cd68cae-eecf-43c3-ae91-36c27b2c3a6e.png)